kafka/error.go

```diff
@@ -40,8 +40,9 @@ func newError(code C.rd_kafka_resp_err_t) (err Error) {
 	return Error{ErrorCode(code), "", false}
 }
 
-func newGoError(code ErrorCode) (err Error) {
-	return Error{code, "", false}
+// NewError creates a new Error with specified ErrorCode and fatal status.
+func NewError(code ErrorCode, fatal bool) (err Error) {
```

Comment:
Let's expose the errstr in this method as well so it can be used for all erroring purposes.

Sorry for all the late ideas ;)

Reply:
No prob :)